Cargo.toml

```diff
@@ -126,10 +126,17 @@ starknet-ff = { version = "0.3.3", default-features = false }
 poseidon_hash = { default-features = false, package = "poseidon", git = "https://github.com/EvolveArt/poseidon-rs", branch = "feature/no-std-refractor", features = [
   "alloc",
 ] }
-blockifier = { git = "https://github.com/keep-starknet-strange/blockifier", branch = "no_std-support", default-features = false }
+blockifier = { git = "https://github.com/apoorvsadana/blockifier", branch = "no_std-support", default-features = false }
```

Comment:
this has been merged you can switch back to the official blockifier

Reply:
yep, I have made the change locally, will push it soon, thanks for informing!

Reply:
fixed